### PR TITLE
delete list of speaker error messages (closes #2237)

### LINKED ISSUE #2237: Delete multiple speakers – How to handle error
If you want to delete multiple speakers, you can send an array with ids. How to handle an error here  (e. g. an invalid speaker id)?

a) Delete nobody? Then we have to add `with transaction.atomic():` to the viewset method.
b) Delete all valid speakers? Then we have to add `break` to the viewset method.
c) Something else ...?

### PR DESCRIPTION
**Pull request by matakuka**
If there are more than 4 removed, there will be an error message with numbers, 
else all removed users will be listed in the message

### CODE REVIEW

**Comment by matakuka** [comment]
Change as discussed: Now the name will only be displayed if it is 1 user, else only the number will be shown.
rebased and ready for review @normanjaeckel

File: tests/integration/agenda/test_viewset.py [hunk 1]
```diff
@@ -195,13 +195,13 @@ def test_remove_someone_else_not_on_list(self):
         response = self.client.delete(
             reverse('item-manage-speaker', args=[self.item.pk]),
             {'speaker': '1'})
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 200)
```
Comment (normanjaeckel) [inline tests/integration/agenda/test_viewset.py]: Please add a test line that the correct message is retrieved.

File: tests/integration/agenda/test_viewset.py [hunk 2]
```diff
@@ -195,13 +195,13 @@ def test_remove_someone_else_not_on_list(self):
...
 
     def test_remove_someone_else_invalid_data(self):
         response = self.client.delete(
             reverse('item-manage-speaker', args=[self.item.pk]),
             {'speaker': 'invalid'})
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 200)
```
Comment (normanjaeckel) [inline tests/integration/agenda/test_viewset.py]: Please add a test line that the correct message is retrieved.

File: tests/integration/agenda/test_viewset.py [hunk 3]
```diff
@@ -195,13 +196,17 @@ def test_remove_someone_else_not_on_list(self):
         response = self.client.delete(
             reverse('item-manage-speaker', args=[self.item.pk]),
             {'speaker': '1'})
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content.decode('UTF-8'), '{"detail":"' +
+                         ugettext('No speakers have been removed from the list of speakers.' + '"}'))
```
Comment (normanjaeckel) [inline tests/integration/agenda/test_viewset.py]: This ugettext looks strange. The `}` is inside the function call.

File: tests/integration/agenda/test_viewset.py [hunk 4]
```diff
@@ -195,13 +196,17 @@ def test_remove_someone_else_not_on_list(self):
...
 
     def test_remove_someone_else_invalid_data(self):
         response = self.client.delete(
             reverse('item-manage-speaker', args=[self.item.pk]),
             {'speaker': 'invalid'})
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content.decode('UTF-8'), '{"detail":"' +
+                         ugettext('No speakers have been removed from the list of speakers.' + '"}'))
```
Comment (normanjaeckel) [inline tests/integration/agenda/test_viewset.py]: Here the same.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Please review again.

**Pull request merged**